### PR TITLE
Add ethereum to cli signer

### PR DESCRIPTION
**Pull request by joelamouche**
tackles #219

### CODE REVIEW

File: packages/api-cli/src/api.ts [hunk 1]
```diff
@@ -163,14 +170,17 @@ async function getCallInfo (): Promise<CallInfo> {
...
-          ? (result as Codec).toHuman()
-          : result
-      }, null, 2)
-    ),
+    log: (result: SubmittableResult | Codec | ApiCallFn): void =>
+      console.log(
+        JSON.stringify(
+          {
+            // eslint-disable-next-line @typescript-eslint/unbound-method
+            [method]: isFunction((result as Codec).toHuman) ? (result as Codec).toHuman() : result
```
Comment (jacogr) [inline packages/api-cli/src/api.ts:178]:
```suggestion
            [method]: isFunction((result as Codec).toHuman)
              ? (result as Codec).toHuman()
              : result
```

File: packages/api-cli/src/api.ts [hunk 2]
```diff
@@ -163,14 +170,17 @@ async function getCallInfo (): Promise<CallInfo> {
...
+          },
+          null,
+          2
+        )
```
Comment (jacogr) [inline packages/api-cli/src/api.ts:182]: Formatting here is quite weird, not sure why.

Comment (joelamouche) [reply packages/api-cli/src/api.ts:182]: Me neither. The linter doesn't complain

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**